### ISSUE TITLE
[Q] - dragging all geometries

### ISSUE THREAD

**Issue by AndrejGajdos** [geoman-io/leaflet-geoman#777]
Thanks for maintaining this plugin. It's very helpful.

If you drag a geometry, is there a way to drag/move all geometries together?

**Comment by Falke-Design** [comment]
There is no built in function for this, but you can try to use this sub-library: https://github.com/Falke-Design/PMDrag

All layers in the same layergroup can dragged together.

**Comment by AndrejGajdos** [comment]
@Falke-Design thank you. I will try it out and let you know.

**Comment by Falke-Design** [comment]
I don't think that I will do this. 
You can copy the script into your src folder and add with `import './yourSrcPath/pmDrag'`

**Comment by AndrejGajdos** [comment]
@Falke-Design I am trying to enable dragging and editing mode at the same time, but I guess this is not possible with Geoman, right?

**Comment by Falke-Design** [comment]
Add this to your code: 
```
    map.on('pm:drag',(e)=> {
      var layer = e.layer || e.target;
      if(layer.pm) {
        if(layer.pm._initMarkers) {
          layer.pm._initMarkers();
        }
        if(layer.pm.applyOptions){
          layer.pm.applyOptions();
        }
      }
    });


    var editDragEnabled = false;
    var actions = ['finishMode'];
    var afterClick = ()=>{
      if(!editDragEnabled) {
        var layers = L.PM.Utils.findLayers(map);
        layers.forEach((layer) => {
          layer.pm.enableLayerDrag();
          layer.pm.enable();
        })
        editDragEnabled = true;
      }else{
        // enable another mode to disable it then
        map.pm.enableGlobalDragMode();
        map.pm.disableGlobalDragMode();
        editDragEnabled = false;
      }
    }
    map.pm.Toolbar.createCustomControl({ name: "DragEdit", block: "custom", title: "Edit and Drag Layers", afterClick, actions: actions, toggle: true });
```

**Comment by AndrejGajdos** [comment]
@Falke-Design awesome, thank you :)

**Comment by Falke-Design** [comment]
will be implemented with #835
